var/spack/repos/builtin/packages/rocm-gdb/package.py

```diff
@@ -28,6 +28,8 @@ class RocmGdb(AutotoolsPackage):
     depends_on('expat', type='build')
     depends_on('python', type='build')
     depends_on('zlib', type='link')
+    depends_on('babeltrace@1.2.4', type='link')
```

Comment:
Is this the only version of babeltrace that works? If not, I would remove the version specification.

Reply:
Yes, its a hard dependency.